types/styletron-react/index.d.ts

```diff
@@ -0,0 +1,271 @@
+// Type definitions for styletron-react 5.0
+// Project: https://github.com/styletron/styletron
+// Definitions by: Eric Taylor <https://github.com/erictaylor>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.1
+
+import * as React from 'react';
+import { driver, StandardEngine, StyleObject } from 'styletron-standard';
+
+export { StyleObject, StandardEngine };
```

Comment:
Is it okay to re-export types from `styletron-standard` here? Or should this be avoided in favor of using the types directly for their defs?

Reply:
I think it is fine for convenience. Maybe the maintainers of styletron-standard have an opinion?